scss/detail.scss

```diff
@@ -1,57 +1,139 @@
-.detail__wrapper {
-  align-items: center;
-  display: flex;
-  flex-direction: column;
-  flex-grow: 1;
-  justify-content: center;
+.grid__wrapper {
+  display: grid;
+  grid-template: 1fr / 1fr;
+  height: 100%;
+  position: absolute;
```

Comment:
Why have absolute positioning? This element is a body child, can you try without absolute?

Reply:
I tried to remove la position property, but I did not get the result I wanted. While, when I use **position: absolute** property I can fill the full height of the body.